packages/iov-ethereum/src/serialization.ts

```diff
@@ -394,12 +292,22 @@ export class Serialization {
   }
 
   private static getErc20Token(
-    unsigned: SendTransaction,
+    unsigned: UnsignedTransaction,
     erc20Tokens: ReadonlyMap<TokenTicker, Erc20Options>,
   ): Erc20Options {
-    const erc20Token = erc20Tokens.get(unsigned.amount.tokenTicker);
+    let erc20Token: Erc20Options | undefined;
+    let ticker: string = "";
```

Comment:
This can be `let ticker: string;`. TypeScript understands that it is set in all non-throwing code paths and will error if you forget to set it somewhere.

Reply:
Nice, it forced me to initialise it before I added the `throw`.